client/modules/IDE/pages/IDEViewMobile.jsx

```diff
@@ -33,16 +79,217 @@ Screen.propTypes = {
   children: PropTypes.node.isRequired
 };
 
-export default () => (
-  <Screen>
-    <Header><h1>Mobile View</h1></Header>
+const isUserOwner = ({ project, user }) => (project.owner && project.owner.id === user.id);
 
+const IDEViewMobile = (props) => {
+  // const
+  const {
+    preferences, ide, editorAccessibility, project, updateLintMessage, clearLintMessage, selectedFile, updateFileContent, files, closeEditorOptions, showEditorOptions, showKeyboardShortcutModal, setUnsavedChanges, startRefreshSketch, stopSketch, expandSidebar, collapseSidebar, clearConsole, console, showRuntimeErrorWarning, hideRuntimeErrorWarning
+  } = props;
 
-    <h3>
-      <br />This page is under construction.
-      <br /><Link to="/">Click here</Link> to return to the regular editor
-    </h3>
+  const [tmController, setTmController] = useState(null);
 
-    <Footer><h1>Bottom Bar</h1></Footer>
-  </Screen>
-);
+  return (
+    <Screen>
+      <Header>
+        <div>
+          <h2>{project.name}</h2>
+          <h3>{selectedFile.name}</h3>
+        </div>
+        <Icon href="/">
```

Comment:
You should import `CloseIcon` from `client/common/icons.jsx`, and wrap it in a `<Link>` or `<button>` depending on what you want it to do. And then, because this icon has no text but performs an action, it should have the attribute `aria-label="close header"` (or whatever the alt text should be.

Reply:
Ok, will do that!